### PR TITLE
Adding service fabric hackathon HOL

### PR DESCRIPTION
**Pull request by jonocairns**

### CODE REVIEW

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 1]
```diff
@@ -0,0 +1,373 @@
...
+
+![](<media/azure9.png>)
+
+After it has successfully deployed you should see the following on the Azure dashboard.
+
+![](<media/azure10.png>)
+
+**Step 9.** Select the Hackathon cluster, then select 'Custom fabric settings'. Add a new value for the **EseStore** with the parameter **MaxCursors** and the value **65536**. This is to ensure our node modules folder will be able to be registered correctly. Service fabric currently has a hard limit on the amount of files and will timeout if this value is not increased. *Ideally you should have a build task that minifies and concatenates all your vendor dependencies*. This may take some time to complete as it needs to propagate across the scale set.
+
+![](<media/max-cursors.png>)
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: Change this to a full click sequence showing the two clicks needed to get teh form and the form please

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 2]
```diff
@@ -0,0 +1,373 @@
...
+```powershell
+Connect-ServiceFabricCluster [your-server-cluster-address].[location].cloudapp.azure.com:19000
+```
+
+Note: ensure the address does not have https:// - eg. `hackathon.westcentralus.cloudapp.azure.com:19000`
+
+**Step 12.** Now we want to upload the application package to the cluster.
+
+```powershell
+Copy-ServiceFabricApplicationPackage -ApplicationPackagePath 'C:\dev\release' -ImageStoreConnectionString 'fabric:imagestore' -ApplicationPackagePathInImageStore 'Apps\Hackathon' -TimeoutSec 600
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: Change the ApplicationPackagePath parameter value to match the example used above of `C:\release`

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 3]
```diff
@@ -0,0 +1,373 @@
...
+
+**Step 14.** Now we want to create a new Service Fabric application.
+
+```powershell
+New-ServiceFabricApplication -ApplicationName 'fabric:/Hackathon' -ApplicationTypeName 'NodeAppType' -ApplicationTypeVersion 1.0
+```
+
+**Step 15.** Go back to the Service Fabric dashboard and check to see if the application has deployed successfully.
+
+![](<media/azure11.png>)
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: This is a repeated screenshot. Did you mean to show something else here? If it's supposed to be the dashboard then an image without the callout for the service fabric explorer url would be better.

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 4]
```diff
@@ -0,0 +1,373 @@
...
+
+
```
Comment (gavinbarron) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: How do I know that the site is working??? We should at a minimum be having them launch the site

File: docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md [hunk 5]
```diff
@@ -0,0 +1,373 @@
...
+Service uptime is one of the most important metrics for any application. Users are quick to switch if they cannot have access 24/7. Azure Service Fabric is a platform designed to make it easy to package, deploy and scale reliable applications. You have been tasked with improving availability for the hackathon starter app. 
+
+**Prerequisites**
+
+- An active Azure subscription
+
+- Service Fabric SDK
+
+- NodeJS 6.0+
```
Comment (jonocairns) [inline docs/HOL-Fault_Injection_for_Service_Fabric_with_Hackathon_Starter/README.md]: I just pulled it from the hackathon starter prereqs page -> https://github.com/sahat/hackathon-starter#prerequisites

**Comment by bernarden** [comment]
Closing due to a project being finished long time ago.

**Pull request closed without merging**